src/libraries/System.Formats.Tar/tests/TarReader/TarReader.File.Tests.cs

```diff
@@ -784,13 +539,30 @@ private void Verify_Archive_Fifo(PosixTarEntry fifo, IReadOnlyDictionary<string,
 
             if (fifo is PaxTarEntry pax)
             {
-                // TODO: Check ext attrs https://github.com/dotnet/runtime/issues/68230
+                VerifyExtendedAttributes(pax);
             }
             else if (fifo is GnuTarEntry gnu)
             {
-                Assert.True(gnu.AccessTime >= DateTimeOffset.UnixEpoch);
-                Assert.True(gnu.ChangeTime >= DateTimeOffset.UnixEpoch);
+                VerifyGnuFields(gnu);
             }
         }
+
+        private void VerifyExtendedAttributes(PaxTarEntry pax)
+        {
+            Assert.NotNull(pax.ExtendedAttributes);
+            Assert.Equal(TarEntryFormat.Pax, pax.Format);
+            Assert.True(pax.ExtendedAttributes.Count() >= 3); // Expect to at least collect mtime, ctime and atime
```

Comment:
For all of the places that you are asserting with `>` or `<`, consider using either Assert.InRange or AsserExtensions.GreaterThan(OrEqualTo).

That way, if the test fails, you don't get

```
Assertion Failed.
  Expected: True
    Actual: False
```

but instead get

```
Expected 2 to be greater than or equal to 3.
```

Automatically you know more... what the bad value was.  (Since AssertExtensions.GreaterThanOrEqualTo is (actual, minimum) and Assert.Equal is (expected, actual), I'd avoid the OrEqualTo versions and just use GreaterThan (or LessThan), to avoid your test being backwards)

Reply:
I changed all the places doing `Assert.True` with `>` or `>=` to instead use `AssertSuggestions.GreaterThan` or `GreaterThanOrEqualTo`.